doc/source/users_guide/index.rst

```diff
@@ -51,110 +50,204 @@ This example shows how you can display a mesh:
-    velocity_vector.display("window-4")
+    pathlines_window = GraphicsWindow()
+    pathlines_window.add_graphics(pathlines)
+    pathlines_window.show()
 
 Plot operations
 ---------------
-Examples follow for plot operations that PyFluent-Visualization
-supports.
+Plot operations that PyFluent-Visualization supports:
 
 Display plot
 ~~~~~~~~~~~~
-This example shows how you can display the XY plot:
 
-.. code:: python
+.. code-block:: python
+
+    from ansys.fluent.visualization import XYPlot
 
-    plots_session_1 = Plots(solver_session)
-    xy_plot = plots_session_1.XYPlots["xy-plot"]
-    xy_plot.surfaces = ["outlet"]
-    xy_plot.y_axis_function = "temperature"
-    xy_plot.plot("window-5")
+    xy_plot = XYPlot(
+        solver=solver_session,
+        surfaces=["outlet"],
+        y_axis_function="temperature",
+    )
+    xy_plot_window = GraphicsWindow()
+    xy_plot_window.add_graphics(xy_plot)
+    xy_plot_window.show()
 
 Display solution residual plot
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-This example shows how you can display the solution residual plot:
 
-.. code:: python
+.. code-block:: python
 
+    from ansys.fluent.visualization import Monitor
 
-    matplotlib_plots1 = Plots(solver_session)
-    residual = matplotlib_plots1.Monitors["residual"]
+    residual = Monitor(solver=solver_session)
     residual.monitor_set_name = "residual"
-    residual.plot("window-6")
+    monitor_window = GraphicsWindow()
+    monitor_window.add_graphics(residual)
+    monitor_window.show()
 
 Display solution monitors plot
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-This example shows how you can display the solution monitors plot:
 
-.. code:: python
+.. code-block:: python
 
-    solver_session.tui.solve.initialize.hyb_initialization()
-    solver_session.tui.solve.set.number_of_iterations(50)
-    solver_session.tui.solve.iterate()
-    solver_session.monitors_manager.get_monitor_set_names()
-    matplotlib_plots1 = Plots(solver_session)
-    mass_bal_rplot = matplotlib_plots1.Monitors["mass-bal-rplot"]
+    solver_session.settings.solution.initialization.hybrid_initialize()
+    solver_session.settings.solution.run_calculation.iterate(iter_count=50)
+
+    mass_bal_rplot = Monitor(solver=solver_session)
     mass_bal_rplot.monitor_set_name = "mass-bal-rplot"
-    mass_bal_rplot.plot("window-7")
+    monitor_window = GraphicsWindow(grid=(1, 2))
+    monitor_window.add_graphics(mass_bal_rplot, position=(0, 0))
+
+    point_vel_rplot = Monitor(solver=solver_session, monitor_set_name="point-vel-rplot")
+    monitor_window.add_graphics(point_vel_rplot, position=(0, 1))
+    monitor_window.show()
+
+Interactive Graphics
+--------------------
+The ``GraphicsWindow`` class provides for managing and directly interacting
+with the graphics window. By registering the window with ``EventsManager``, you
+can update graphics during run time and create animations.
+
+The following example updates the graphics window (contour_window, xy_plot_window and
```

Comment:
```suggestion
The following example updates the graphics window (contour_window, xy_plot_window, and
```
Serial (or what is also known as Oxford comma). Always use a comma before the "and" or "or" in a list of three or more items.

Reply:
@PipKat @prmukherj @mkundu1 I believe that we need to change our config to enforce this style.